client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -34,34 +34,28 @@ class DesignTypeWithStoreStep extends Component {
 
 	getChoices() {
 		const { translate } = this.props;
-
-		if ( abtest( 'signupStepOneCopyChanges' ) === 'modified' ) {
-			// Note: Don't make this translatable because it's only visible to English-language users
-			return [
-				{ type: 'blog',
-					label: 'a blog',
-					description: 'To share your ideas, stories, and photographs with your followers.',
-					image: <BlogImage /> },
-				{ type: 'page',
-					label: 'a website',
-					description: 'To promote your business, organization, or brand and connect with your audience.',
-					image: <PageImage /> },
-				{ type: 'grid',
-					label: 'a portfolio',
-					description: 'To present your creative projects in a visual showcase.',
-					image: <GridImage /> },
-				{ type: 'store',
-					label: 'an online store',
-					description: 'To sell your products or services and accept payments.',
-					image: <StoreImage /> },
-			];
-		}
+		const blogText = translate( 'To share your ideas, stories, and photographs with your followers.' ), // eslint-disable-line max-len
+			siteText = translate( 'To promote your business, organization, or brand and connect with your audience.' ), // eslint-disable-line max-len
+			gridText = translate( 'To present your creative projects in a visual showcase.' ),
```

Comment:
it is more conventional to have separate declarations per line.

maybe this kind of format would sort out the eslint issue:

```
const varName = translate(
  '...'
);
```